src/constants.js

```diff
@@ -0,0 +1,2 @@
+export const JWT_ISSUER = "carrot-pinning-proxy";
```

Comment:
Should we rename this to something else? Since it's a new service.

Reply:
Oh my bad. A copy paste thing. Fixed in f46382ed35e8ac7d8961e24d9e4798d8023616bb